pkg/cmd/root.go

```diff
@@ -138,7 +141,8 @@ func init() {
 	rootCmd.PersistentFlags().Bool("no-read-env", false, "do not read enclave config from the environment")
 	rootCmd.PersistentFlags().String("scope", ".", "the directory to scope your config to")
 	rootCmd.PersistentFlags().String("configuration", configuration.UserConfigFile, "config file")
-	rootCmd.PersistentFlags().Bool("json", false, "output json")
-	rootCmd.PersistentFlags().Bool("debug", false, "output additional information when encountering errors")
-	rootCmd.PersistentFlags().Bool("silent", false, "disable output of info messages")
+	rootCmd.PersistentFlags().Bool("json", utils.OutputJSON, "output json")
+	rootCmd.PersistentFlags().Bool("debug", utils.Debug, "output additional information")
+	rootCmd.PersistentFlags().Bool("print-config", false, "output active configuration")
```

Comment:
I also considered naming this `--debug-config`, which would mean that the flag would only work when specified with `--debug` (i.e. `--debug --debug-config`). Lmk your thoughts.